### PR TITLE
Populate status message

### PR DESCRIPTION
**Pull request by marzagao**
This PR adds logic for each provider implementation to populate the field `statusMessage` on job status, which will used during error conditions to store a more user friendly error message.

### CODE REVIEW

File: provider/elastictranscoder/aws.go [hunk 1]
```diff
@@ -285,9 +285,18 @@ func (p *awsProvider) JobStatus(job *db.Job) (*provider.JobStatus, error) {
 			Width:    aws.Int64Value(resp.Job.Input.DetectedProperties.Width),
 		}
 	}
+	statusMessage := ""
+	if len(resp.Job.Outputs) > 0 {
+		statusMessage = aws.StringValue(resp.Job.Outputs[0].StatusDetail)
+		if strings.Contains(statusMessage, ":") {
+			errorMessage := strings.Split(statusMessage, ":")[1]
```
Comment (flavioribeiro) [inline provider/elastictranscoder/aws.go]: do we need to split it? maybe we could pass the entire error?

Comment (marzagao) [inline provider/elastictranscoder/aws.go]: The entire error looks like this:

`4000 b107230c-1f2a-4e8d-a2d9-3709612c91fb: Amazon Elastic Transcoder could not interpret the media file.`

So with the split and trimp above I get:

`Amazon Elastic Transcoder could not interpret the media file`

Comment (flavioribeiro) [inline provider/elastictranscoder/aws.go]: nice!

Comment (fsouza) [inline provider/elastictranscoder/aws.go]: I wonder if there's a risk of having a ``:`` in the statusMessage, in which case we should probably use ``SplitN(statusMessage, ":", 2)``.

**Comment by marzagao** [comment]
Oh, Flávio already merged it...

**Comment by marzagao** [comment]
Let me revert. :)

**Pull request merged**